packages/components/src/MultiSelect/MultiSelect.tsx

```diff
@@ -40,6 +40,10 @@ interface MultiSelectProps {
   readonly size?: "small" | "large";
 }
 
+// TODO: Remove me. This is a comment in an underused component to convince lerna to do a major version update when we asked nicely.
+// Please remove me soon.
+export const newVariable = "This is not in use.";
```

Comment:
It may be worth exporting this out of the index.ts here to trick lerna more
https://github.com/GetJobber/atlantis/blob/0271a7feaab02cf84782cb6cf4afc10a40fe5501/packages/components/src/MultiSelect/index.ts